test/suites/test_dml.py

```diff
@@ -31,17 +38,64 @@ def setUpClass(self):
             parts = {2, 'num', 3, 'str'},
             unique = false})
         """.replace('\n', ' '))
+
         self.space_created = self.adm("box.schema.create_space('space_2')")
         self.adm("""
         box.space['space_2']:create_index('primary', {
             type = 'hash',
             parts = {1, 'num'},
             unique = true})
         """.replace('\n', ' '))
+
+        self.adm("box.schema.create_space('space_str')")
+        self.adm("""
+        box.space['space_str']:create_index('primary', {
+            type = 'tree',
+            parts = {1, 'str'},
+            unique = true})
+        """.replace('\n', ' '))
+
+        self.adm("box.schema.create_space('space_varbin')")
+        self.adm("""
+        box.space['space_varbin']:create_index('primary', {
+            type = 'tree',
+            parts = {1, 'varbinary'},
+            unique = true})
+        """.replace('\n', ' '))
+        self.adm("""
+        buffer = require('buffer')
+        ffi = require('ffi')
+
+        function encode_bin(bytes)
+            local tmpbuf = buffer.ibuf()
+            local p = tmpbuf:alloc(3 + #bytes)
+            p[0] = 0x91
+            p[1] = 0xC4
+            p[2] = #bytes
+            for i, c in pairs(bytes) do
+                p[i + 3 - 1] = c
+            end
+            return tmpbuf
+        end
+
+        function bintuple_insert(space, bytes)
+            local tmpbuf = encode_bin(bytes)
+            ffi.cdef[[
+                int box_insert(uint32_t space_id, const char *tuple, const char *tuple_end, box_tuple_t **result);
+            ]]
+            ffi.C.box_insert(space.id, tmpbuf.rpos, tmpbuf.wpos, nil)
+        end
+        """)
```

Comment:
https://automated-testing.info/t/v-chem-raznicza-mezhdu-ispolzovaniem-setup-i-setupclass/14929/2
```
setUpClass()
A class method called before tests in an individual class run. setUpClass is called with the class as the only argument and must be decorated as a classmethod():
```

```
setUp()
Method called to prepare the test fixture. This is called immediately before calling the test method; other than AssertionError or SkipTest, any exception raised by this method will be considered an error rather than a test failure. The default implementation does nothing.
```

So I think setting it up in `setUpClass` is legitimate. Moreover, any other preparations like space setup and module requires are made here, so I wrote it here too for consistency.